input/metadata/properties/default.json

```
@@ -5071,329 +5157,269 @@
         }
       }
     },
-    "manure_handler_configs": {
+    "handler": {
       "type": "array",
+      "description": "A list of all configurations for handlers.",
       "properties": {
         "type": "object",
+        "description": "Configuration for a handler processor.",
         "name": {
           "type": "string",
-          "description": "Unique identifier to reference this manure handler config in manure management scenario(s). This value should be unique and cannot match the name of any other manure processor."
+          "description": "The unique identifier of the handler processor. Cannot match the name of any other manure processor."
         },
-        "manure_handler_type": {
+        "processor_type": {
           "type": "string",
-          "description": "The type of manure handler (method for cleaning barn alleyways).",
-          "pattern": "^(flush system|alley scraper|manual scraping|tillage|harrowing)$"
-        },
-        "cleaning_water_use_rate": {
-          "type": "number",
-          "description": "The rate of water use (L per animal per day, total of fresh and/or recycled water) by a specified manure handler (liters)",
-          "minimum": 0
-        },
-        "minutes_per_cleaning": {
-          "type": "number",
-          "description": "The length of a specified manure handler's cleaning cycle (minutes).",
-          "minimum": 0
+          "description": "The type of handler used.",
+          "pattern": "^(AlleyScraper|ManualScraper|FlushSystem|ParlorCleaningHandler)$"
         },
-        "cleanings_per_day": {
+        "cleaning_water_use_amount": {
           "type": "number",
-          "description": "The number of cleanings a specified manure handler performs per day.",
-          "minimum": 0
-        },
-        "daily_tillage_frequency": {
-          "type": "number",
-          "description": "Applicable to 'tillage'-type handlers; the number of times per day that a specified compost bedded pack barn pack is tilled.",
-          "minimum": 0
+          "description": "Amount of cleaning water used per animal per day (L/animal/day).",
+          "minimum": 0.0,
+          "maximum": 100.0,
+          "default": 0.0
         },
         "cleaning_water_recycle_fraction": {
           "type": "number",
-          "description": "The fraction (0.XX) of cleaning water used by a specified handler that is from recycled (not fresh) water sources (unitless).",
+          "description": "Fraction of cleaning water that is from recycled (not fresh) water sources (unitless).",
           "minimum": 0.0,
-          "maximum": 1.0
+          "maximum": 1.0,
+          "default": 0.8
+        },
+        "use_parlor_flush": {
+          "type": "bool",
+          "description": "Indicator for if a flush system is used in the milking parlor and/or holding area. This flush water may be partially or fully recycle is in addition to routine parlor cleaning that utilizes only fresh water."
```

```suggestion
          "description": "This input is applicable only to parlor cleaning handlers. Indicator for if a flush system is used in the milking parlor and/or holding area. This flush water may be partially or fully recycled and is in addition to routine parlor cleaning that utilizes only fresh water.",
          "default": false
```
Adding a default of 'false' here so that we can potentially omit this input from non-parlor-cleaning handler types in the example input JSON file (in Niko's daily process PR), since it's only applicable to parlor cleaning handlers